align/schema/constraint.py

```diff
@@ -613,7 +613,7 @@ def check(self, checker):
 
         if self.max_height is not None:
             assert self.max_height >= 0, f'Boundary:max_height should be greater than zero {self.max_height}'
-            checker.append(checker.cast(bvar.ury-bvar.lly, float) <= 1000*self.max_height, float) # in nanometer
```

Comment:
@soneryaldiz Don't know what the float is for. It wasn't on the max_width version.

Reply:
This is a mistake. Thank you for the fix.